Makefile

```diff
@@ -100,7 +102,12 @@ print-infra-secrets: read-tf-config install-fetch-config set-azure-account
 	bin/fetch_config.rb -s azure-key-vault-secret:${key_vault_name}/${key_vault_infra_secret_name} -f yaml
 
 get-cluster-credentials: read-cluster-config set-azure-account ## make <config> get-cluster-credentials [ENVIRONMENT=<clusterX>]
-	az aks get-credentials --overwrite-existing -g ${AZURE_RESOURCE_PREFIX}-tsc-${CLUSTER_SHORT}-rg -n ${AZURE_RESOURCE_PREFIX}-tsc-${CLUSTER}-aks
+ifeq ($(GITHUB_ACTIONS),)
```

Comment:
Can we use the same approach as in the [template](https://github.com/DFE-Digital/teacher-services-cloud/blob/main/templates/new_service/Makefile#L120-L129) for consistency?